kimchi/src/tests/foreign_field_add.rs

```diff
@@ -1453,3 +1456,106 @@ where
 
     (Ok(()), witness)
 }
+
+#[test]
+// Finalization test
+fn test_ffadd_finalization() {
+    // Includes a row to store value 1
+    let num_inputs = 1;
+    let operation = &[FFOps::Add];
+    let modulus = BigUint::from_bytes_be(&[
+        0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFE, 0xFF, 0xFF, 0xFF, 0xFF,
+        0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFE, 0xFF, 0xFF,
+        0xFC, 0x2F,
+    ]);
+
+    // circuit
```

Comment:
You should comment the circuit layout here for reference below when creating copy constraints.